File: src/Workflow.Web.Service/Workflow/app/model/common/Employee.js
Ext.define('Workflow.model.common.Employee', {
    extend: 'Workflow.model.Base',
    idProperty: 'Id',
    fields: [
        "Id","DeptId","LoginName","EmpNo","JobTitle","DisplayName","Email"
        , "Telephone", "MobilePhone", "HomePhone", "IpPhone", "Address"
        , {
            name: "HiredDate",
            convert: function (value) { return value ? Ext.Date.format(new Date(value), 'Y-m-d'): value; }
        }
        , "ReportTo", "DeptName", "FirstName", "LastName", "EmpType", "Active"
    ],
    proxy: {
        writer: { writeAllFields: true },
        type: 'rest',
        api: {
            read: Workflow.global.Config.baseUrl + 'api/employee/manual',
            create: Workflow.global.Config.baseUrl + 'api/employee',
            update: Workflow.global.Config.baseUrl + 'api/employee',
            destroy: Workflow.global.Config.baseUrl + 'api/employee'
        },
        actionMethods: {
            read: 'GET',
            destroy: 'DELETE',
            update: 'PUT',
            create: 'POST'
        },
        reader: {
            type: 'json',
            rootProperty: 'Records',
            totalProperty: 'TotalRecords'
        }
    }
});

File: src/Workflow.Domain/Entities/Employee.cs
/**
*@author : Phanny
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Workflow.Domain.Entities.Base;

namespace Workflow.Domain.Entities
{
    public partial class Employee
    {
        public int Id { get; set; }

        public Nullable<int> DeptId { get; set; }

        public string LoginName { get; set; }

        public string EmpNo { get; set; }

        public string JobTitle { get; set; }

        public string DisplayName { get; set; }

        public string Email { get; set; }

        public string Telephone { get; set; }

        public string MobilePhone { get; set; }

        public string HomePhone { get; set; }

        public string IpPhone { get; set; }

        public string Address { get; set; }

        public Nullable<System.DateTime> HiredDate { get; set; }

        public string ReportTo { get; set; }

        public string DeptName { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string EmpType { get; set; }

        public Nullable<bool> Active { get; set; }

        public virtual Department Department { get; set; }
    }
}
